packages/fx-core/resource/unify-config-change-logs.md

```diff
@@ -19,6 +19,8 @@ Your existing `localSettings.json`, `manifest.local.template.json`, and `manifes
 ## Know what you need to do
 Since Teams Toolkit will use `manifest.remote.template.json` as a single manifest template file after the upgrade, if you have customized the `manifest.local.template.json`, you will need to update the `manifest.template.json` to include your change.
 
+For more information you can visit [Unify config help wiki](https://github.com/OfficeDev/TeamsFx/wiki/Upgrade-to-support-unify-config).
```

Comment:
use aka link, we can link to anchor of the page later.

Reply:
Fixed.